osx/KA-Lite_Uninstall.tool

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+
+# Override any funny stuff from the user.
+export PATH="/bin:/usr/bin:/sbin:/usr/sbin:$PATH"
+
+#----------------------------------------------------------------------
+# Global Variables
+#----------------------------------------------------------------------
+KALITE_MONITOR_APP="/Applications/KA-Lite-Monitor.app"
+KALITE_APP="/Applications/KA-Lite/KA-Lite.app"
```

Comment:
@arceduardvincent Am I correct that we have a `/Applications/KA-Lite/` folder because we now include an uninstall tool?